backend/store/setting.go

```diff
@@ -280,3 +280,105 @@ func patchSettingImpl(ctx context.Context, tx *Tx, patch *api.SettingPatch) (*se
 	}
 	return &settingRaw, nil
 }
+
+// FindSettingMessage is the message for finding setting.
+type FindSettingMessage struct {
+	Name api.SettingName
+}
+
+// UpdateSettingMessage is the message for updating setting.
+type UpdateSettingMessage struct {
+	Name  api.SettingName
+	Value string
+}
+
+// SettingMessage is the message of setting.
+type SettingMessage struct {
+	UID         int
+	Name        api.SettingName
+	Value       string
+	Description string
+}
+
+// GetSettingV2 returns the setting by name.
+func (s *Store) GetSettingV2(ctx context.Context, find *FindSettingMessage) (*SettingMessage, error) {
+	tx, err := s.db.BeginTx(ctx, &sql.TxOptions{ReadOnly: true})
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to begin transaction")
+	}
+	defer tx.Rollback()
+
+	setting, err := getSettingV2Impl(ctx, tx, find)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := tx.Commit(); err != nil {
+		return nil, errors.Wrap(err, "failed to commit transaction")
+	}
+	return setting, nil
+}
+
+func getSettingV2Impl(ctx context.Context, tx *Tx, find *FindSettingMessage) (*SettingMessage, error) {
```

Comment:
Inline getSettingV2Impl(). We only need the Impl when necessary. Same for updateSettingV2Impl().